extension/httpfs/src/s3fs.cpp

```diff
@@ -17,6 +17,7 @@ void AWSEnvironmentCredentialsProvider::setOptionValue(main::ClientContext* cont
     auto accessKeyID = context->getEnvVariable(S3_ACCESS_KEY_ENV_VAR);
     auto secretAccessKey = context->getEnvVariable(S3_SECRET_KEY_ENV_VAR);
     auto endpoint = context->getEnvVariable(S3_ENDPOINT_ENV_VAR);
+    auto urlStyle = context->getEnvVariable(S3_URL_STYLE_ENV_VAR);
```

Comment:
what is urlStyle? Do we have tests covering this?

Reply:
Yes, see test in: 
```
-CASE AttachS3Duckdb
-STATEMENT load extension "${KUZU_ROOT_DIRECTORY}/extension/duckdb/build/libduckdb.kuzu_extension"
---- ok
-STATEMENT CALL duckdb_s3_access_key_id='${UW_S3_ACCESS_KEY_ID}'
---- ok
-STATEMENT CALL duckdb_s3_secret_access_key='${UW_S3_SECRET_ACCESS_KEY}'
---- ok
-STATEMENT CALL duckdb_s3_endpoint='rgw.cs.uwaterloo.ca'
---- ok
-STATEMENT CALL duckdb_s3_url_style='path'
---- ok
-STATEMENT CALL duckdb_s3_region='US'
---- ok
```